frontend/src/views/WheresMyGene/components/HeatMap/components/Chart/index.tsx

```diff
@@ -221,6 +235,9 @@ export default memo(function Chart({
 
         setChartProps(result);
 
+        allChartProps[tissue] = result;
+        setAllChartProps(allChartProps);
```

Comment:
since we now have `AllChartProps` state lifted, maybe we can remove `const [chartProps, setChartProps] = useState<ChartProps | null>(null);` in Chart component and just get `chartProps` from parent?

e.g.,

```ts
<Chart chartProps={allChartProps[tissue]} />
```